librz/core/cmd_descs/cmd_info.yaml

```diff
@@ -447,8 +448,9 @@ commands:
       - RZ_OUTPUT_MODE_QUIETEST
     args: []
   - name: izz
-    cname: cmd_info_whole_strings(see also str.search.*)
+    cname: cmd_info_whole_strings
     summary: List strings in the whole binary
+    description: see also str.search.*
```

Comment:
Same here: `Lists the strings of the whole binary; the behavior can be modified via str.search.*`